starlark/src/values/mod.rs

```diff
@@ -95,18 +95,35 @@ use std::fmt;
 use std::marker;
 use std::rc::Rc;
 
+#[derive(Clone)]
```

Comment:
nit: a few documentation line on what ValueInner goal is would be good here. E.g.:

// ValueInner wrap the actual value or a memory pointer to the actual value for complex type.